projectile.el

```diff
@@ -3538,8 +3547,8 @@ a manual COMMAND-TYPE command is created with
                                   :test "sbt test"
                                   :test-suffix "Spec")
 
-(projectile-register-project-type 'mill '("build.sc")
-                                  :project-file "build.sc"
+(projectile-register-project-type 'mill #'projectile-mill-project-p
+                                  :project-file '("build.sc" "build.mill")
```

Comment:
Won't this require some code changes or it was already accounted for?

Reply:
As in allowing lists? - If so I believe it should already be accounted for: https://github.com/bbatsov/projectile/blob/db52cf48b6cc3af892ba699ec13c61334bd39d1b/projectile.el#L2934

Reply:
Ah, ok. I just noticed you updated the docstrings, but there were no other changes, so it seemed prudent to ask about this. :-)

Reply:
Ah yeap, probably should have pointed that out, sorry!